chatterino.pro

```diff
@@ -509,7 +509,7 @@ linux:isEmpty(PREFIX) {
 }
 
 linux {
-    desktop.files = resources/chatterino.desktop
+    desktop.files = resources/com.chatterino.chatterino2.desktop
```

Comment:
The "application id" we use for other OSes has been `com.chatterino.chatterino` - I think we should stay consistent. Please update this in the other changes you've made too.